features/file-system-access.yml

```diff
@@ -4,8 +4,18 @@ spec: https://wicg.github.io/file-system-access/
 caniuse: native-filesystem-api
 group: file-system
 compat_features:
+  - api.FileSystemDirectoryHandle.@@asyncIterator
+  - api.FileSystemFileHandle.createWritable
   - api.FileSystemHandle.queryPermission
   - api.FileSystemHandle.requestPermission
+  - api.FileSystemSyncAccessHandle.close.sync_version
+  - api.FileSystemSyncAccessHandle.flush.sync_version
+  - api.FileSystemSyncAccessHandle.getSize.sync_version
+  - api.FileSystemSyncAccessHandle.truncate.sync_version
```

Comment:
After some more reading, it seems like these `FileSystemSyncAccessHandle` things are just for OPFS.

MDN says this: 

> This class is only accessible inside dedicated [Web Workers](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API) [...] for files within the [origin private file system](https://developer.mozilla.org/en-US/docs/Web/API/File_System_API/Origin_private_file_system), which is not visible to end-users.

So I believe we should move these 4 keys to the OPFS feature instead.